tests/test_preflight_check.py

```diff
@@ -153,9 +156,9 @@ async def test_client_config(patched_logger, mock_responses):
     mock_es_info(mock_responses)
```

Comment:
Nit: same as above, I think a test with multiple connectors would be good to have here as well.

Reply:
I think the difference is that, we should test prelight here, not how the config is parsed.

Reply:
That makes sense to me 👍🏻